test/jdk/java/lang/RuntimeTests/exec/ExecCommand.java

```diff
@@ -68,163 +84,187 @@ public static String unquote(String str)
+        @Override public void checkWrite(String file) {}      // TestNG writes files
         @Override public void checkDelete(String file) {}
-        @Override public void checkRead(String file) {}
+        @Override public void checkPropertiesAccess() {}
+        @Override public void checkPropertyAccess(String name) {}  // TestNG reads properties
     }
 
-    // Parameters for the Runtime.exec calls
-    private static final String TEST_RTE_ARG[] = {
-        "cmd /C dir > dirOut.txt",
-        "cmd /C dir > \".\\Program Files\\dirOut.txt\"",
-        ".\\Program Files\\do.cmd",
-        "\".\\Program Files\\doNot.cmd\" arg",
-        "\".\\Program Files\\do.cmd\" arg",
-        // compatibility
-        "\".\\Program.cmd\" arg",
-        ".\\Program.cmd arg",
-    };
-
-    private static final String doCmdCopy[] = {
+    private static final List<String> doCmdCopy = List.of(
         ".\\Program.cmd",
         ".\\Program Files\\doNot.cmd",
-        ".\\Program Files\\do.cmd",
-    };
-
-    // Golden image for results
-    private static final String TEST_RTE_GI[][] = {
-                    //Def Legacy mode, Enforced mode, Set Legacy mode, Set Legacy mode & SM
-        // [cmd /C dir > dirOut.txt]
-        new String[]{"Success",
-                     "IOException",  // [cmd /C dir ">" dirOut.txt] no redirection
-                     "Success",
-                     "IOException"}, //SM - no legacy mode, bad command
-
-        // [cmd /C dir > ".\Program Files\dirOut.txt"]
-        new String[]{"Success",
-                     "IOException",  // [cmd /C dir ">" ".\Program Files\dirOut.txt"] no redirection
-                     "Success",
-                     "IOException"}, //SM - no legacy mode, bad command
-
-        // [.\Program File\do.cmd]
-        new String[]{"Success",
-                     "IOException",  // [.\Program] not found
-                     "Success",
-                     "IOException"}, //SM - no legacy mode [.\Program] - OK
-
-        // [".\Program File\doNot.cmd" arg]
-        new String[]{"Success",
-                     "Success",
-                     "Success",
-                     "AccessControlException"}, //SM   - [".\Program] - OK,
-                                 //     [.\\Program Files\\doNot.cmd] - Fail
-
-        // [".\Program File\do.cmd" arg]
-        // AccessControlException
-        new String[]{"Success",
-                     "Success",
-                     "Success",
-                     "Success"}, //SM - [".\Program] - OK,
-                                 //     [.\\Program Files\\do.cmd] - OK
-
-        // compatibility
-        new String[]{"Success", "Success", "Success", "Success"}, //[".\Program.cmd"]
-        new String[]{"Success", "Success", "Success", "Success"}  //[.\Program.cmd]
-    };
+        ".\\Program Files\\do.cmd");
 
     private static void deleteOut(String path) {
         try {
             Files.delete(FileSystems.getDefault().getPath(path));
```

Comment:
More of a curious question, is there a reason you couldn't use Files::deleteIfExists or Path.of() instead of FileSystems.getDefault....

Reply:
That code is pre-existing.  Your suggestion can completely replace the deleteOut method.  Tnx